src/ripple/protocol/impl/TxFormats.cpp

```diff
@@ -84,6 +84,13 @@ TxFormats::TxFormats ()
     add ("TicketCancel", ttTICKET_CANCEL)
         << SOElement (sfTicketID,            SOE_REQUIRED)
         ;
+
+    // The SignerEntries are optional because a SignerList is deleted by
+    // setting the SignerQuorum to zero and omitting SignerEntries.
+    add ("SignerListSet", ttSIGNER_LIST_SET)
```

Comment:
I agree.  Arthur has argued that the `Set` transaction types should all have the `Set` at the end for consistency.  But for files and constants I picked names that were consistent with those in the codebase neighborhood, which meant that the `Set` went at the front for those.  It felt awkward at the time (as it still does).

However when I look at
https://github.com/scottschurr/rippled/blob/m-of-n/src/ripple/protocol/impl/TxFormats.cpp#L55
it appears that names of transactions may already be inconsistent on this point.  It's true that I'm seeing `AccountSet` and `TrustSet`, but I'm also seeing `SetRegularKey` and `SetFee`.  I think `SetSignerList` would be easier for an English reader to comprehend and would be consistent with the file and object names.  But I'm not ready to go to the mat over this one.